application/modules/home/views/side_ci.php

```diff
@@ -0,0 +1,5 @@
+<h3 class="box-title"><i class="fa fa-building"></i> <?php echo $exhibition['name']; ?> に加盟している会社情報編集</h3>
+<!--
+End of file side_ci.php
+Location: ./modules/home/views/side_ci.php
```

Comment:
__[code_sniffer]__
* No comment block marks the end of file instead of the closing PHP tag. Please add a comment block containing only "End of file side_ci.php".
* No comment block marks the end of file instead of the closing PHP tag. Please add a comment block containing only "Location: ./modules/home/views/side_ci.php".

<!-- Comment by [SideCI](https://sideci.com) -->